src/script.js

```diff
@@ -262,42 +250,42 @@ function showOutputs(cost_e_cb, cost_m_cb, cost_ex_m_cb, cost_agg_new,
 
 // Izračunaj stroške
```

Comment:
try to break down functin to smaller parts. So create seperate function try to seperate each operation into seperate functions.